### PR TITLE
mine changes

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request

as asked nerfs the mines and overlays em above everything

## Why It's Good For The Game

mines are no visible with trash. and not so fraggy

## Changelog
:cl:Fernandos33
tweak: mines are now overlayed like traps
balance: frag from excels mines where lowered
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/mine.dm [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
 	matter_reagents = list("fuel" = 40)
+	layer = ABOVE_MOB_LAYER //should fix all layering problems? or am i crazy stupid and understood it wrong
 	var/prob_explode = 100
 
 	//var/obj/item/device/assembly_holder/detonator = null
 
 	var/fragment_type = /obj/item/projectile/bullet/pellet/fragment/weak
 	var/spread_radius = 4
-	var/num_fragments = 30
+	var/num_fragments = 15
```
Comment (SpadesSlick) [inline code/game/objects/items/weapons/mine.dm:16]: That's too much of a nerf in one go. Put it to 25 and let's see if it works.

File: code/game/objects/items/weapons/mine.dm [hunk 2]
```diff
@@ -6,13 +6,14 @@
 	w_class = ITEM_SIZE_BULKY
 	matter = list(MATERIAL_STEEL = 30)
 	matter_reagents = list("fuel" = 40)
+	layer = ABOVE_MOB_LAYER //should fix all layering problems? or am i crazy stupid and understood it wrong
```
Comment (SpadesSlick) [inline code/game/objects/items/weapons/mine.dm:9]: Mines above mobs is just wrong.

File: code/game/objects/items/weapons/mine.dm (hunk 1 above)
Comment (fernandos13) [reply code/game/objects/items/weapons/mine.dm:16]: can do

File: code/game/objects/items/weapons/mine.dm (hunk 2 above)
Comment (fernandos13) [reply code/game/objects/items/weapons/mine.dm:9]: sure can change it too
above turf then?

**Comment by fernandos13** [comment]
changes made

**Comment by fernandos13** [comment]
abov OBJ this should do

**Pull request merged**